### DISCUSSION TITLE
Schema的dependencies写法错误

### DISCUSSION THREAD

**Discussion by cheng-xiaofeng** [alibaba/formily#3058] [RFC]
![image](https://user-images.githubusercontent.com/33592850/164364747-349f40cc-c6d8-44f5-bd46-512c85abb5c5.png)
我跟着官网写的dependencies: ['viewtype'](我本地的名称), 运行会报错, 但是加上一个点dependencies: ['.viewtype'], 这样就没事了?官网的例子写错了吗

**Comment by janryWang** [comment]
前面加点代表相对路径，不带点是绝对路径
